src/main/jhipster/src/main/webapp/app/shared/column-filter/column-filter.component.html

```diff
@@ -85,7 +85,7 @@
                         <!-- TODO dynamic custom popover filter -->
                     </ng-template>
                     <button *ngIf="filter.type !== FILTER_TYPES.MODAL"
-                            type="button" class="btn btn-info" [ngbPopover]="popover" [placement]="['bottom', 'right', 'left']" [autoClose]="'outside'"
+                            type="button" class="btn btn-info" [ngbPopover]="popover" [placement]="['bottom', 'right']" [autoClose]="'outside'"
```

Comment:
The `left` part was dealing with the scenario where there are many filters added, and one of them reaches the right border of the screen, in in that case opening to the right or bottom could leave the popover partially hidden.
Though I guess that's a less likely scenario, so probably we can leave with this workaround for the moment, though I wonder why this could be happening, maybe a bug in the library that don't detect the visible boundaries correctly.